### ISSUE TITLE
Wordnet synsets query problem 

### ISSUE THREAD

**Issue by ndvbd** [nltk/nltk#2441]
Doing:
`print wn.synsets("tablesssssssssssssssss")`
returns results (it shouldn't, as there is no such word in English),
But if we use the Wordnet online API here:
`http://wordnetweb.princeton.edu/perl/webwn`
for the same word, we get no results (correct).
Why is this problem?

**Comment by alvations** [comment]
It's because the WordNet `wn.synsets()` functions tries to lemmatize using the `moprhy` lemmatizer before checking for synsets. Using the new API, you can get the strict behavior of not  lemmatizing by default by specifying `use_morphy=False` 

```python
>>> from wn import WordNet
>>> wn = WordNet()
wn.synsets("tablessssssssssssssss">>> wn.synsets("tablessssssssssssssss")
[Synset('table.n.01'), Synset('table.n.02'), Synset('table.n.03'), Synset('mesa.n.02'), Synset('table.n.05'), Synset('board.n.04'), Synset('postpone.v.01'), Synset('table.v.02')]
>>> wn.synsets("tablessssssssssssssss", use_morphy=False)
[]
```

**Comment by ndvbd** [comment]
I upgraded from nltk 3.2.5 to 3.4.5 but still don't have the use_morphy argument. 
Why?

**Comment by ekaf** [comment]
Yes @ndvbd,, the "use_morphy" argument is not even in the latest NLTK version, though was proposed in [issue 18](https://github.com/nltk/wordnet/pull/18#issue-513042920l), and sounds like a good idea...

**Comment by ekaf** [comment]
Actually, the problem is not whether or not to use morphy, but rather to prevent morphy  from recursively stripping the same suffix many times. PR #3225 fixes it.

**Comment by ekaf** [comment]
Fixed in #3225.
